apps/minifront/src/components/swap/swap-form/estimate-button.tsx

```diff
@@ -0,0 +1,46 @@
+import { buttonVariants } from '@penumbra-zone/ui/components/ui/button';
+import {
+  Tooltip,
+  TooltipProvider,
+  TooltipTrigger,
+  TooltipContent,
+} from '@penumbra-zone/ui/components/ui/tooltip';
+import { cn } from '@penumbra-zone/ui/lib/utils';
+
+export const EstimateButton = ({
```

Comment:
Moved over from `simulate-swap-button` below, and added props to be passed in rather than accessing the state directly from inside this component (since this component is now shared between the `instantSwap` and `dutchAuction` slices).